InventoryBundleProduct/Model/IsProductSalableCondition/IsBundleSaleableCondition.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryBundleProduct\Model\IsProductSalableCondition;
+
+use Magento\Bundle\Model\Product\Type;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\InventoryBundleProduct\Model\GetBundleProductStockStatus;
+use Magento\InventorySalesApi\Api\IsProductSalableInterface;
+
+class IsBundleSaleableCondition implements IsProductSalableInterface
```

Comment:
Dock block?

Reply:
@Stepa4man, it is not required if the class name describes its purpose well.